tendrl-common.spec

```diff
@@ -0,0 +1,57 @@
+Name: tendrl-common
+Version: 0.0.1
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Common Module for all Bridges
+Source0: %{name}-%{version}.tar.gz
+License: LGPLv2+
+URL: https://github.com/Tendrl/bridge_common
+
+BuildRequires: pytest
+BuildRequires: python2-devel
+BuildRequires: python-mock
+BuildRequires: python-six
+BuildRequires: systemd
+BuildRequires: python-yaml
+
+Requires: python-dateutil
+Requires: python-dns
+Requires: python-etcd
+Requires: systemd-python
+Requires: python-urllib3
+
+%description
+Common python module usable by all Tendrl SDS Bridges
+
+%prep
+%setup
+
+# Remove bundled egg-info
+rm -rf %{name}.egg-info
+
+%build
+%{__python} setup.py build
+
+%install
+%{__python} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+install -m  644 --directory $RPM_BUILD_ROOT%{_var}/log/tendrl/common
+install -Dm 644 etc/tendrl/tendrl.conf.sample $RPM_BUILD_ROOT%{_datadir}/tendrl/commons/tendrl.conf.sample
+install -Dm 644 etc/samples/logging.yaml.timedrotation.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/common_logging_yaml
+install -Dm 644 etc/samples/*.sample $RPM_BUILD_ROOT%{_datadir}/tendrl/commons/.
+
+%check
+#py.test -v tendrl/common/tests
```

Comment:
It is better to run the py.test command here, but tack on `|| :` at the end so that it is non-fatal. This way we get to see the test output in the RPM build logs, even if the tests are not entirely stable yet.